rust/src/lib.rs

```diff
@@ -175,6 +175,13 @@ pub mod ffi {
     }
 
     // core.rs
+    #[derive(Debug)]
+    enum RefspecType {
+        Ostree,
+        Checksum,
+        Container,
+    }
```

Comment:
It'd be more churn, but would it be even better to have:

```
struct Refspec {
  typ: RefspecType,
  value: String
}
```
?

Reply:
Hmm yeah I think that could make sense as an accessor on treefile since a lot of callsites immediately then classify it. If you don't mind, I'll try to make this part of https://github.com/coreos/rpm-ostree/pull/3545 instead.